doc/pages/faq.asciidoc

```diff
@@ -34,13 +34,29 @@ This variable should be set by the terminal emulator, and not overridden with
 an arbitrary value, otherwise it might interfere with general UI rendering on
 the terminal's window.
 
+Currently, Kakoune relies on "palette switching": redefining the standard
+256-color palette for true 24-bit RGB color support.
+If you're using a tool that doesn't support "palette switching", colors will
+be slightly off: they are being rounded down to values supported by the
+regular color palette by Kakoune.
+
+== Why are colors unreadable in `konsole`/`QTermWidget` or `st`?
+
+Even if the `TERM` and `COLORTERM` environment variables are set correctly, most
```

Comment:
It's probably not worth mentioning `COLORTERM` here; I don't know of any applications that actually use it, certainly Kakoune doesn't.